components/pagination/collections/paging_collection.js

```diff
@@ -377,6 +373,9 @@
                 }
             }
         });
+
+        var PagingCollectionProto = PagingCollection.prototype;
```

Comment:
I figured that since the `PagingCollectionProto` is the prototype of the `PagingCollection`, it made sense to declare it afterwards.